assume/strategies/flexable.py

```diff
@@ -345,8 +345,8 @@ def calculate_bids(
             min_power[start] = unit.calculate_ramp(
                 op_time, previous_power, min_power[start], current_power
             )
-            bid_quantity = min_power[start] - previous_power
-            if bid_quantity >= 0:
+            bid_quantity = previous_power - min_power[start]
+            if bid_quantity <= 0:
```

Comment:
This is a bid for negative control reserve. There was just a small bug that was bidding negative value for supply instead of positive values. So the negative CRM results were always empty. This simply reverses the sign, nothing much to it